app/views/kondos/index.html.erb

```diff
@@ -1,15 +1,13 @@
 <div class="container">
   <div class="row justify-content-center">
-    <div class="col-6">
       <% @kondos.all.each do |kondo| %>
```

Comment:
I was just wondering, what is the difference between using @kondos.all.each and @kondos.each when you iterate?

Reply:
Oh I think carl did this because he didn't define .all at the controller, I just added the card for this page haha

Reply:
lol, `@kondos.all.each` I'll fix this 👍🏼

Reply:
got it here: minor change, should be a free merge: https://github.com/spioupiou/is-kondo/pull/31/files